### ISSUE TITLE
NIxOS Virtualbox OVA: Setup instructions do not work

### ISSUE THREAD

**Issue by codygman** [divnix/devos#134]
I can't paste in the commands now, but following the manual from a clean OVA I had a few issues. Later I'll post out issues from each step.

## Your Environment
https://channels.nixos.org/nixos-20.09/latest-nixos-x86_64-linux.ova

**Comment by nrdxp** [comment]
Can we close this?

**Comment by codygman** [comment]
Maybe? I'll try doing this process again after work and see if it's now fixed. I can't remember what the issue was previously.

**Comment by blaggacao** [comment]
Maybe a good avenue to satisfy the presumed underlaying use case would be to expose the lib.nixosSyatem VM from within the `flk` command, so that we could do `flk vm Hostname`, similar.tp how we do `flk iso Hostname`.

**Comment by Narice** [comment]
yes the `flk vm Hostname` proposition would be awesome! that's exactly what I wanted scrolling through the issues. is it possible to do something like that as of now?
I would like to build a new system using devos and thus I would like to test my configuration on a vm before committing to it ;)
Btw, I didn't really understand what was the issue at hand here so I hope my comment makes sense in this context.

**Comment by Pacman99** [comment]
You can currently do `nix build .#nixosConfigurations.<hostname>.config.system.build.vm`. It should be pretty simple to add this as a flk command.

**Comment by Narice** [comment]
Yes! that worked! thank you so much @Pacman99
For reference, here is the command to run the vm afterwards:
```
./result/bin/run-<hostname>-vm
```
